src/libraries/System.Private.Xml.Linq/src/System/Xml/Linq/XElement.cs

```diff
@@ -464,7 +467,7 @@ public IEnumerable<XElement> DescendantsAndSelf(XName name)
         /// </summary>
         public XNamespace GetDefaultNamespace()
```

Comment:
`public IEnumerable<XElement> DescendantsAndSelf(XName name)` should take nullable `name`